test/python/circuit/test_parameters.py

```diff
@@ -939,6 +939,13 @@ def test_rebinding_instruction_copy(self):
                 self.assertEqual(expected1, output1)
                 self.assertEqual(expected2, output2)
 
+    def test_sign_of_parameter(self):
+        """Test returning the sign of the value of the parameter"""
+
+        b = Parameter("phi")
+        sign_of_parameter = b.sign()
+        print("sign of Parameter is: ", sign_of_parameter)
```

Comment:
It's best not to use `print` in a test. Usually, tests contain some form of `self.assert...` to assert some condition that should be true if the code is working correctly. Here, I think you could something like `self.assertEqual(sign_of_parameter.assign(b, -3), -1)`.